index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="styles.css">
+        <title>Practice Website</title>
+  </head>
+
+  <body>
+    <div class="top">
+      <div class="header">
+        <div class="header-logo">Header Logo</div>
+  
+        <div class="links">
```

Comment:
Using a [nav] semantic element here would look good as well.

_Whenever you find a opportunity to use a semantic element you should try your best to use it. They were meant for their specific uses so using a div instead of a semantic element may raise some questions that could be avoided easily_